Sources/Filters+Strings+Boolean.swift

```diff
@@ -0,0 +1,54 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import Stencil
+
+extension Filters {
+  enum Strings {
```

Comment:
Mmmh. There's no particular reason why the `enum` would be declared in `+Boolean` and `extension` be used in the other files… But is it worth having a `Filters+Strings.swift` which would only declare an empty `enum Strings{}` just so all `Filters+Strings+Whatever.swift` subsequent file are consistent and all use `extension Filters.Strings`?